iamchrissmith/uncorked
# Stream follows

#### What does this PR do?
Integrates Stream.io for user following venue.

#### Where should the reviewer start?
`controller/follows_controller.rb`

#### How should this be manually tested?
Log in as a user, follow a venue.  In a private window, login as a different user and follow the same venue.  Switch back to your first user.  You should see the other user's follow in your feed.

#### Any background context you want to provide?
#### What are the relevant tickets?
[148850273](https://www.pivotaltracker.com/story/show/148850273)  
[149045185](https://www.pivotaltracker.com/story/show/149045185)  
[148684209](https://www.pivotaltracker.com/story/show/148684209)  

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/1408372/28396667-291a66a0-6cba-11e7-8871-4b6c99dcd698.png)

#### Questions:
  - Do Migrations Need to be ran? **Yes**
  - Do Environment Variables need to be set? **Yes** `STREAM_KEY` and `STREAM_SECRET`
  - Any other deploy steps?
